clients/src/test/java/org/apache/kafka/clients/consumer/internals/OffsetsRequestManagerTest.java

```diff
@@ -86,7 +86,6 @@
 import static org.mockito.Mockito.when;
 
 public class OffsetsRequestManagerTest {
-
```

Comment:
Nit: I find it's better to avoid changes in unrelated files, even if minor

Reply:
My bad

Reply:
Could you please revert this change?